recipes/mpfun90/meta.yaml

```diff
@@ -0,0 +1,83 @@
+{% set name = "mpfun90" %}
+{% set version = "2024-12-05" | replace('-', '.') %}
```

Comment:
```suggestion
{% set version = "2024.12.05" %}
```

Reply:
@xhochy If there are maintenance changes to the source repo (https://github.com/APN-Pucky/mpfun90) the tags there are going to be `YYYY-MM-DD`. If I switch to using `YYYY.MM.DD` then won't conda-forge not automatically pick these up? This is why I'm using the Jinja commands currently.

Reply:
The logic of the bot is:
1. Take the version
2. Bump it a bit to look for new releases
3. Check whether the URL produces new results

In the end, it will look at the output of the jinja variable which is with dots. It isn't working on the plain string here.